packages/chat-core-aws-connect/src/infra/ChatCoreAwsConnectImpl.ts

```diff
@@ -0,0 +1,133 @@
+import { ChatCoreAwsConnect } from "../models/ChatCoreAwsConnect";
+import { MessageRequest, MessageResponse } from "@yext/chat-core";
+import { AwsConnectEvent } from "../models/AwsConnectEvent";
+import { EventCallback } from "../models/EventCallback";
+import "amazon-connect-chatjs";
+
+// TODO: Remove this type once the integration details are added to the MessageResponse type in chat-core
```

Comment:
that should be added to chat-core already, can we update this?

Reply:
Whoops, this comment is misleading. Meant to say "once region is added" because that's not present yet but is returned from ChatApi. Updated the comment to reflect